orchestration/hca_orchestration/tests/e2e/test_copy_project.py

```diff
@@ -1,7 +1,12 @@
 import logging
 
 import pytest
-from dagster import execute_pipeline, in_process_executor, PipelineExecutionResult, ResourceDefinition
+from dagster import (
+    PipelineExecutionResult,
```

Comment:
Discussed via slack but also noting here: I think this test and the associated copy_project pipeline can be removed.